### ISSUE TITLE
rbmviz charts are not updating side panel

### ISSUE THREAD

**Issue by mattroumaya** [Gilead-BioStats/gsmApp#103]
When a site on chart is selected, it is no longer updating the side-panel. 

Need to trace previous work to see what's causing the error.

**Comment by taylorrodgers** [comment]
Hey @mattroumaya, I wanted to try to do more to isolate the issue. Basically, the following is not always triggering properly:
```
snapshot$lCharts[["kri0001"]]$scatterJS
snapshot$lCharts[["kri0001"]]$scatterJS$x$bHideDropdown = TRUE
snapshot$lCharts[["kri0001"]]$scatterJS
```
This is true in the Viewer as it is the Shiny session.

As far as I can tell, this is **not** an R version issue nor a Shiny version issue, but it does appear to be related to a package. I haven't isolated the package yet, as the usual offenders (`shiny`, `shinyjs`, `htmltools`, `htmlwidgets`) doesn't trigger any change.

However, if you install the `gsmApp` package and agree to install the updates for every package, this issue **does** goes away.

**Comment by mattroumaya** [comment]
Closed via #107
